src/tuvx/interface_util.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2023-2024 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+//
+// This file contains hepler functions for the internal TUV-x interface.
+#include "interface_util.hpp"
+
+namespace musica
+{
+  String InternalCreateString(const char *str)
+  {
+    String string;
+    string.size_ = strlen(str);
+    string.value_ = new char[string.size_ + 1];
+    strcpy(string.value_, str);
+    return string;
+  }
```

Comment:
It seems that the behavior of this code is duplicated in `src/util.cpp` under the `CreateString` function. Is there a specific reason for having this implementation? Does the 'Internal' prefix have any significance?

Reply:
good catch! I didn't realize i was recreating something that already existed. I removed this function